tests/BenchmarkDotNet.IntegrationTests/CoreRtTests.cs

```diff
@@ -17,6 +17,9 @@ public class CoreRtTests : BenchmarkTestExecutor
         [Fact]
         public void CoreRtIsSupported()
         {
+            if (IntPtr.Size == sizeof(int)) // CoreRT does not support 32bit yet
```

Comment:
We use this `IntPtr.Size` trick too often. I guess it makes sense to introduce a single API and use it everywhere.